File: Project Euler 7 (Python)/Project Euler 7.py
import math

def isPrime(number):
	if number < 2:
		return False
	
	for i in range(2, int(math.sqrt(number)) + 1):
		if number % i == 0:
			return False
	return True

found = 0
i = 0
while True:
	i+=1
	if isPrime(i):
		found+=1
		if found >= 10001:
			print(i)
			break

File: JavaScript/Project Euler 7 (JavaScript)/Project Euler 7.js
function isPrime(num) {
	for(let i = 2; i < num; i++)
		if(num % i === 0) return false;
	return num !== 1;
}

let i = 0;
let found = 0;

while(true) {
	i++;
	if(isPrime(i)) {
		found++;
		if (found >= 10001) {
			console.log(i);
			break;
		}
	}
}